### ISSUE TITLE
Config dir detection in flex

### ISSUE THREAD

**Issue by DidierLmn** [EasyCorp/easy-deploy-bundle#39]
Due to issue #16 the config directory detection was changed to: 

> The detection should be: use the new config dir if Symfony 4.x or (etc/ exists and app/config/ doesn't exist).

However the config directory has been named `/config` again, instead of `/etc` (see [this issue](https://github.com/symfony/flex/issues/56)) so this bundle looks in the wrong directory and throws the following error:

> None of the usual Symfony config dirs exist in the application. Create one of these dirs before continuing: "%s/etc" or "%s/app/config".

**Comment by onokje** [comment]
also, and perhaps slightly unreleated: The default log dir in symfony 4 is var/log and not var/logs.

**Comment by kaizokou** [comment]
This pull request seems to fix it : https://github.com/EasyCorp/easy-deploy-bundle/pull/40
It would be so great if it can be merged.

**Comment by TheKassaK** [comment]
So what ? A solution for this problem today ?

**Comment by kaizokou** [comment]
You need to fork the the PR I mentioned above 
Then you will use it in your composer.json
This is a temporary fix until it's merged.


```
    "require-dev": {
        "easycorp/easy-deploy-bundle": "dev-master",
    },
    "repositories": [
        {
            "type": "vcs",
            "url":  "git@github.com:kaizokou/easy-deploy-bundle.git"
        }
    ],
```

**Comment by javiereguiluz** [comment]
Fixed by #40. Thanks!
